### DISCUSSION TITLE
Amount of chia on my wallet does not add up!? help please

### DISCUSSION THREAD

**Discussion by eusergiopt** [Chia-Network/chia-blockchain#7999] [Support]
Hi everyone. I need some help since I can not understand what is going on.
I’m solo farming and never joined any pool.
All rewards went fine except one particular reward(1.749999999) that went to a different receiving address. this reward has never been reflected on my balance. I have no idea ehat is causing this. Can someone help and explain what is going on? 
I have tried to read but I can not find any similar situation. basically the coin appeas on my wallet but I can not access it! I wonder if this is a bug since numbers in the wallet should surely add up to the total..
![Screen Shot 08-08-21 at 04 34 PM](https://user-images.githubusercontent.com/88676467/128740436-19a970b0-47a3-462c-940e-907cd22371ee.PNG)
![Screen Shot 08-08-21 at 06 09 PM](https://user-images.githubusercontent.com/88676467/128740456-fe8ac87c-a6ea-4522-9299-0f12e9ef2a56.PNG)

many thanks in advance

**Comment by eusergiopt** [comment]
Above I show all wallet transactions

normal receive address:
xch1fl47wt9hwdmyk8kwygs95ksp5msw3y4ngcksy998tnjzfxaytjgqaxx7qd
everything expect the 1.7499999999
second receive that is not accounted:
xch1vxxgtvux9ajmhrf4txgnl3nepfpnqrq2ddxcthckpsyvz4jgl3aqggrhgj

the 1.7499999999 is actually there but how can I access this?

**Comment by loppefaaret** [comment]
to me, it looks like change from a mojo transaction (the smallest amount in chia) - often from a plot NFT command:
your wallet most likely chose a "1.75 coin" and split this into 0.000000000001 and 1.749999999999...
the 1.749999999999 you see incoming also technically left your wallet, although only for them to be returned imidiately after.
it is like going to a shop, wanting to buy a bottle of water for 2.45 - you pay with a "20 coin", the cashier will take the 20, and hand you back 17.55...

**Reply by eusergiopt** [reply]
Thanks for your comment. I'm not quite sure I understand since there is 1.74999..9 missing and it never came back. Also how can my wallet just do things that I'm not ordering it to do? Is that really possible?
However it is curious and intriguing to see that my last farmed reward contained 0.25000..00001, which complements the 1.74999999999. I really don't understand what is all this!!

**Reply by keliew** [reply]
It's about the coin model that chia uses. See https://github.com/Chia-Network/chia-blockchain/wiki/FAQ#what-is-the-coin-model

You can access it as how you normal would. It's not lost. It's just presented differently under a different wallet _address_ but on the same wallet _key_.

**Reply by eusergiopt** [reply]
Thanks for you reply and help Keliew. It is interesting to lear that. However, to me fo pratical purposes it is a lost coin since the balance does not reflect it, despite the wallet keeping track of its existence. Is it possible to actually spend or use somehow that coin, considering it is missing from the balance? how would that be possible? regards

**Reply by keliew** [reply]
which balance? i think you should distinguish between a wallet (address) and a key.
https://github.com/Chia-Network/chia-blockchain/wiki/Chia-Keys-Management

**Reply by eusergiopt** [reply]
Hi thanks for you reply. I should one have one address since I have opted for option one only one key for evrything. I have never (at least intentionally) set up other addresses to receive rewards. I also have not created plots for pools since I'm solo farming.
I'm using all default configurations. Anyway would I be able to access this second address where the coin is with my menemonic key? I mean how can I actually access this 1.7499...9 coin? why did a particular coin went to a different adress and then all resumes as normal. Sorry for the perhaps naive questions but I don't quite understand the details of how the wallet and address work, it all seems a bit confusing to me.

**Reply by loppefaaret** [reply]
to protect your privacy, it is standard in the crypto world to create new addresses under the same wallet / private key after an address has been used to receive a transaction - places such as "farmer reward target address" and "payout address" under plotNFT's are static - but most other transactions will be directed into new addresses, that includes any "change" you get from splitting a coin into fragments.

**Reply by keliew** [reply]
A single key (mnemonic seed) has (automagically) multiple wallet addresses. When you transact, that wallet address can change. It has no bearing effect to your total balance. And there's no _need_ to access the wallet address to make the transaction, you have to define (usually defaults to `1`) which key you use to make a transaction.

Not exactly sure what your concern is...

If you want to understand the details, you should read up those links...but I don't see how it would benefit you in your transaction needs...

**Reply by eusergiopt** [reply]
> to me, it looks like change from a mojo transaction (the smallest amount in chia) - often from a plot NFT command:
> your wallet most likely chose a "1.75 coin" and split this into 0.000000000001 and 1.749999999999...
> the 1.749999999999 you see incoming also technically left your wallet, although only for them to be returned imidiately after.
> it is like going to a shop, wanting to buy a bottle of water for 2.45 - you pay with a "20 coin", the cashier will take the 20, and hand you back 17.55...

It seems this is what is causing confusion. I just notice a pending change in my wallet corresponding to the 1.7499..9.
I was trying to add incoming to see if my balance was right and things would not add up. so it seems my wallet is spliting coins and doing movements back and forth without me doing anything. see the movement below, and the change of spendable balance down and then up again. I have not actully done any transaction myself.

![Screen Shot 08-12-21 at 03 36 PM](https://user-images.githubusercontent.com/88676467/129216305-a0e2ed18-96cc-44cb-b726-8f71e43350f4.PNG)
these all seems a bit confusing. but in practice perhaps I should just ignore those movements....

**Reply by eusergiopt** [reply]
yes this is it and this happened now that I have created a plot for self pooling. I was ignoring the pool before but when I orderer this pool plot I had a coin splitted and saw these "strange" movements on my wallet! I think I now kind of understand what is going on. perhaps this mojo is necessary to make a NFT plot...

**Reply by keliew** [reply]
https://github.com/Chia-Network/chia-blockchain/wiki/Pooling-User-Guide#step-2-receive-some-xch
It is by design necessary.

**Reply by eusergiopt** [reply]
OK i think I have understood now. I just wonder if for self farming I should just ignore this NFT/pool things implemented all together. is there any advantage of doing this NFT plots for such case. many thanks for evryone that tried to help!

**Reply by keliew** [reply]
https://github.com/Chia-Network/chia-blockchain/wiki/Pooling-User-Guide
The only advantage of having an NFT plot is that you are able to join pools, as detailed in the beginning of that wiki page.

Don't thank me, thank the Wiki and also thank yourself for being able to read. :)
